File: server/keycloak/src/main/java/com/recipefy/keycloak/model/TokenResponse.java
package com.recipefy.keycloak.model;

import com.fasterxml.jackson.databind.PropertyNamingStrategies;
import com.fasterxml.jackson.databind.annotation.JsonNaming;
import lombok.Data;

@Data
@JsonNaming(PropertyNamingStrategies.SnakeCaseStrategy.class)
public class TokenResponse {
    private String accessToken;
    private String refreshToken;
    private long expiresIn;
    private Long refreshExpiresIn;
    private String tokenType;
    private String scope;
}

File: server/api-gateway/src/main/kotlin/com/recipefy/gateway/model/TokenResponse.kt
package com.recipefy.gateway.model

import com.fasterxml.jackson.databind.PropertyNamingStrategies
import com.fasterxml.jackson.databind.annotation.JsonNaming

@JsonNaming(PropertyNamingStrategies.SnakeCaseStrategy::class)
data class TokenResponse(
    val accessToken: String,
    val refreshToken: String,
    val expiresIn: Long,
    val refreshExpiresIn: Long,
    val tokenType: String,
    val scope: String
)
